bug-1671522-pref-assess-change-to-windows-mouse-wheel-scroll-anima-release-82-83.toml

```
@@ -0,0 +1,32 @@
+[experiment]
+end_date = '2020-11-17'
+
+segments = ['regular_users_v3', 'is_new_attributed', 'is_not_new_attributed']
+
+## Segments
+
+[segments.is_new_attributed]
+select_expression = '{{agg_any("is_new_attributed")}}'
+data_source = "new_attributed"

```

I think you can write:

```suggestion
select_expression = '{{agg_any("days_since_created_profile = 0 AND attribution.medium IS NOT NULL")}}'
data_source = "clients_last_seen"
```

and the symmetric thing below